lua/packer/display.lua

```diff
@@ -81,7 +86,7 @@ local display = {}
 local display_mt = {
   --- Check if we have a valid display window
   valid_display = function(self)
-    return not in_headless and self and api.nvim_buf_is_valid(self.buf) and api.nvim_win_is_valid(self.win)
+    return self.interactive and self.win and api.nvim_win_is_valid(self.win)
```

Comment:
Out of curiosity, why does this drop the check that `self.buf` is valid?

Reply:
It was an oversight, sorry about that. Let me fix it

Reply:
Ok pushed a fix! :D

Reply:
Great, thank you!